src/Microsoft.Extensions.Localization.Abstractions/LocalizedString.cs

```diff
@@ -25,7 +25,7 @@ public LocalizedString(string name, string value)
         /// </summary>
         /// <param name="name">The name of the string in the resource it was loaded from.</param>
         /// <param name="value">The actual string.</param>
-        /// <param name="resourceNotFound">Whether the string was found in a resource. Set this to <c>false</c> to indicate an alternate string value was used.</param>
+        /// <param name="resourceNotFound">Whether the string was not found in a resource. Set this to <c>true</c> to indicate an alternate string value was used.</param>
```

Comment:
I asked a couple others and they agree with you, I guess this is more of a personal preference on my part rather than an actual english rule. You're all good to go.